README.md

```
@@ -179,6 +179,35 @@ Docker で複数インスタンスを動かす場合は、ポートをずらし
+# 開発モード（Vite dev server + Tauri WebView）
+bun run tauri:dev
+
+# プロダクションビルド（インストーラー生成）
+bun run tauri:build
+```
+
+> **Windows + Git Bash の場合**: MSVC のビルドツールが PATH に含まれていない場合、
+> Developer Command Prompt for VS 2022 から実行するか、
+> `.bashrc` に `LIB`, `INCLUDE`, `PATH` を設定してください。
+> 詳細は [Issue #49](https://github.com/otomatty/zedi/issues/49) を参照。
+
```

markdownlint-cli2 (0.22.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)